packages/slice-machine/lib/models/common/Environment.ts

```diff
@@ -13,17 +13,22 @@ export interface UpdateVersionInfo {
   updateAvailable: boolean;
 }
 
-export default interface Environment {
+export default interface BackendEnvironment extends FrontEndEnvironment {
   cwd: string;
+  prismicData: PrismicData;
+  client: DefaultClient | FakeClient;
+}
+
+export interface FrontEndEnvironment {
   userConfig: UserConfig;
-  hasConfigFile: boolean;
   repo?: string;
-  prismicData: PrismicData;
   chromatic?: Chromatic;
   updateVersionInfo: UpdateVersionInfo;
   mockConfig: any;
   framework: Framework;
   baseUrl: string;
   hasGeneratedStoriesPath: boolean;
```

Comment:
I think this is unused except on the tests x)
![image](https://user-images.githubusercontent.com/41187899/143853693-69703027-e8bf-476a-8cac-5be27e57ff63.png)

Reply:
Good catch ✅